app/components/molecules/section-info/section-info.styles.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+export interface Props {
+  variant: 'none' | 'internet' | 'events' | 'ingredients' | 'games';
+  children: string;
```

Comment:
isnt `children` a prop already defined by default? - or are we just explicitly doing a definition to be string here? that's ok.

Reply:
Yes! that's right

Reply:
nah https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/basic_type_example make it ReactNode please